src/number_theory/legendre_symbol/add_character.lean

```diff
@@ -63,18 +63,32 @@ The trivial additive character (sending everything to `1`) is `(1 : add_char R R
 @[derive [comm_monoid, inhabited]]
 def add_char : Type (max u v) := (multiplicative R) →* R'
 
-instance add_char.has_coe_to_fun : has_coe_to_fun (add_char R R') (λ x, multiplicative R → R') :=
-monoid_hom.has_coe_to_fun
+end add_char_def
+
```

Comment:
I think you can already enter the `namespace add_char` at this point.

Reply:
I've moved the `namespace` command up a level.